src/service/plugins/share.js

```diff
@@ -196,6 +196,9 @@ const SharePlugin = GObject.registerClass({
                 ];
                 iconName = 'document-save-symbolic';
 
+                let gtk_recent_manager = Gtk.RecentManager.get_default();
```

Comment:
```suggestion
                const gtk_recent_manager = Gtk.RecentManager.get_default();
```

It's not like it's going to change at any point.

Reply:
Heh. In fact, our `eslint` configuration requires this change, so I'm just going to go ahead and apply it.